config/locales/en.yml

```
@@ -13,9 +13,9 @@ en:
             name: privacy policy
             text: To learn more, please read our {privacyPolicy}.
         consent_notice:
-          description: "We collect and process your personal information for the following purposes: {purposes}."
+          description: 'We collect and process your personal information for the following
+            purposes: {purposes}.'
           learn_more: Learn more
```

Hi, I think we should keep the `decline` translation key
```suggestion
          learn_more: Learn more
          decline: Decline
```

---

Then it should be added to ignored unused-key